integrations/oci/sdk/README.md

```diff
@@ -90,6 +90,7 @@ Besides being able to inject OCI SDK APIs, note also that these contracts are in
 * [OciAvailability](runtime/src/main/java/io/helidon/integrations/oci/sdk/runtime/OciAvailability.java) - can be used to determine if the current runtime environment is executing on an OCI compute node.
```

Comment:
I don't see these two classes (OciAvailability and OciRegionProvider) in PR yet.